insights/docs/lessons_learned.md

```diff
@@ -23,4 +23,49 @@ Another problem, QS doesn't have support CMK, which is CMS policy to use. The cu
 More useful links:
 - https://docs.aws.amazon.com/quicksight/latest/user/troubleshoot-athena-insufficient-permissions.html
 
-- https://aws.amazon.com/premiumsupport/knowledge-center/quicksight-deny-policy-allow-bucket/
\ No newline at end of file
+- https://aws.amazon.com/premiumsupport/knowledge-center/quicksight-deny-policy-allow-bucket/
+
+### Make a plan for cost estimation and monitoring
+
+ - Perform cost estimates early in development on a small sample of the data to assess viability on the complete dataset. 
```

Comment:
is the cost linear?

Reply:
it depends on what you are doing. see the next sentence about continuing to assess costs while scaling up though which gets to the point that I believe you are making. it may not be linear or well behaved at all so never stop monitoring and assessing.